custom/conf/app.example.ini

```diff
@@ -1704,6 +1704,12 @@ LEVEL = Info
 ;;
 ;; convert \r\n to \n for Sendmail
 ;SENDMAIL_CONVERT_CRLF = true
+;;
+;; convert links of attached images to inline images. Only for images hosted in this gitea instance.
+;BASE64_EMBED_IMAGES = false
+;;
+;; The maximum size of an image attachment to be embedded in the email.
+;BASE64_EMBED_IMAGES_MAX_SIZE = 5242880
```

Comment:
I guess we do not need to introduce a new option here. There are already too many config options.

A default value like 5M should be good enough for all users. If there would be some special requirements, it won't be late to introduce an option the.

And one more thing, limiting a single file seems not enough. There could be 100 files * 5M , I guess nobody would really like to receive an email with 500M size.

Reply:
> Maybe you can just resize the bigger images to fixed width/height before base64 encode so they will take up much less space?

I think we have to compress the image before base64 it

Reply:
As far as I know, most email providers limit the size of the entire email to 10-30M, but after base64, the attachment will expand by about 1.5 times, and the actual size of the attachment that can be added will be further reduced.